### PR TITLE
DOCS-1128-IA-Admission-controller

### PR DESCRIPTION
**Pull request by bmckercher123**
Clarify text in IA Admission controller.

Product Version(s):
CC 3.17, 3.18, 3.19

Issue:
https://tigera.atlassian.net/browse/DOCS-1128

Link to docs preview:
https://deploy-preview-965--tigera.netlify.app/calico-cloud/image-assurance/install-the-admission-controller

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

File: calico-cloud_versioned_docs/version-3.17/image-assurance/install-the-admission-controller.mdx [hunk 1]
```diff
@@ -61,36 +61,44 @@ Container admission policies are custom Kubernetes resources that allow you to c
...
+As a safety mechanism, we require that you specify the namespaces that the Admission Controller will apply in
+the `IN_NAMESPACE_SELECTOR_KEY` and the `IN_NAMESPACE_SELECTOR_VALUES`. These values configure the Kubernetes API server to send admission requests to our Admission Controller only for resources from relevant namespaces.
 
-   For example, to configure the Kubernetes API server to send admission requests for resources created in any namespace with label key `foo`, and label values either `bar` or `baz`, the variables would be set as follows:
+For example, to configure the Kubernetes API server to send admission requests for
+resources created in any namespace with label key `name`, and label values either `prod` or `critical`, set the variables as follows:
 
    ```bash
-   export IN_NAMESPACE_SELECTOR_KEY="foo" && IN_NAMESPACE_SELECTOR_VALUES="bar baz"
+   export IN_NAMESPACE_SELECTOR_KEY="name" && IN_NAMESPACE_SELECTOR_VALUES="prod critical"
```
Comment (stevegaossou) [inline calico-cloud_versioned_docs/version-3.17/image-assurance/install-the-admission-controller.mdx]: This is a bit of a paranoid view, but is the example of using selector label "critical" here a good idea, given that later on we say that "critical" namespaces like `kube-system` should be avoided. 

I recognize it's actually talking about different things, but just wondering if we can use a different term other than critical for one of the two cases to further differentiate them.

Comment (bmckercher123) [reply calico-cloud_versioned_docs/version-3.17/image-assurance/install-the-admission-controller.mdx]: I'm fine with differentiating. @WilliamTigera What do you suggest I change the "prod critical" to?

Comment (WilliamTigera) [reply calico-cloud_versioned_docs/version-3.17/image-assurance/install-the-admission-controller.mdx]: We should put maybe `staging` as the other label, and change all the other references of `critical` to staging as needed.

Comment (bmckercher123) [reply calico-cloud_versioned_docs/version-3.17/image-assurance/install-the-admission-controller.mdx]: Updated all to "staging".

File: calico-cloud/image-assurance/install-the-admission-controller.mdx [hunk 2]
```diff
@@ -61,36 +61,44 @@ Container admission policies are custom Kubernetes resources that allow you to c
...
-   export IN_NAMESPACE_SELECTOR_KEY="foo" && IN_NAMESPACE_SELECTOR_VALUES="bar baz"
+   export IN_NAMESPACE_SELECTOR_KEY="name" && IN_NAMESPACE_SELECTOR_VALUES="prod staging"
    ```
+Here is the namespace manifest with the staging label and name key.
+
+   ```yaml
+   kind: Namespace
+   apiVersion: v1
+   metadata:
+     name: production
```
Comment (WilliamTigera) [inline calico-cloud/image-assurance/install-the-admission-controller.mdx]: let's change this name here to staging-test

Comment (bmckercher123) [reply calico-cloud/image-assurance/install-the-admission-controller.mdx]: Fixed.

Comment (WilliamTigera) [reply calico-cloud/image-assurance/install-the-admission-controller.mdx]: Sorry, The `name: production` field should also be `name: staging-test`, after that it looks good to me.

Comment (bmckercher123) [reply calico-cloud/image-assurance/install-the-admission-controller.mdx]: Fixed.

**Pull request merged**